### PR TITLE
Hide auto reveal service

### PR DESCRIPTION
**Pull request by fabriziovigevani**

### CODE REVIEW

**Comment by dizzypaty** [comment]
@fabriziovigevani 

My suggestion would be to still show the auto-reveal switch and description in a disabled state accompanied by a label that says "Coming soon". 

Also, here's the copy changes for the warning & info messages: (also added these screens to the Figma file!)



Warning: 

`You must copy or download this code before you can commit your vote. For maximun security, this code will also be saved to your browser’s local storage. You will be asked to confirm it in order to reveal your vote.`



Info:

`This temporary code will be valid to commit and reveal your vote for this dispute only. You will be asked to confirm it in order to reveal your vote.`

**Comment by rperez89** [comment]
> @fabriziovigevani
> 
> My suggestion would be to still show the auto-reveal switch and description in a disabled state accompanied by a label that says "Coming soon".
> 
> Also, here's the copy changes for the warning & info messages: (also added these screens to the Figma file!)
> 
> 
> 
> Warning:
> 
> `You must copy or download this code before you can commit your vote. For maximum safety, this code will also be saved to your browser’s local storage. You will be asked to confirm it in order to reveal your vote.`
> 
> 
> 
> Info:
> 
> `This temporary code will be valid to commit and reveal your vote for this dispute only. You will be asked to confirm it in order to reveal your vote.`

@dizzypaty we didn't talk about `this code will also be saved to your browser’s local storage` so that was not implemented yet

**Comment by dizzypaty** [comment]
> @dizzypaty we didn't talk about `this code will also be saved to your browser’s local storage` so that was not implemented yet

Ooh I thought we did and that we were saving it by default, independently of auto-reveal being on or off. Ok, so if this is not ready yet, we should remove that bit from the `info` copy. 

Warning:

`You must copy or download this code before you can commit your vote. You will be asked to confirm it in order to reveal your vote. Failure to do so might result in a monetary penalty to your account.`

File: src/components/Disputes/panels/CommitPanel.js [hunk 1]
```diff
@@ -259,7 +271,7 @@ const InfoSection = React.memo(function InfoSection({
 }) {
   const content = revealService
     ? 'This temporary code will be valid to commit and reveal your vote for this dispute only. You won’t be required to enter this code unless a problem occur with our services.'
-    : 'You must copy or download this code before you can commit your vote. You’ll be asked to enter it in order to reveal your vote in:'
+    : 'You must copy or download this code before you can commit your vote. You will be asked to confirm it in order to reveal your vote. Failure to do so might result in a monetary penalty to your account.'
```
Comment (sohkai) [inline src/components/Disputes/panels/CommitPanel.js]:
```suggestion
    : 'You must copy or download this code before you can commit your vote. You will be asked to confirm it in order to reveal your vote. Failure to do so will result in a monetary penalty to your account.'
```

**Pull request merged**